docs/hsm.md

```diff
@@ -0,0 +1,105 @@
+# HSM Configuration
```

Comment:
It might be nice to have a single table at the top here somewhere that summarize all HSM assets across both HSM types (offline and SPM) before jumping into the details below.

Reply:
Fixed.